src/static/css/skip-nav.less

```diff
@@ -0,0 +1,24 @@
+#skip-nav {
+    padding: 10px;
```

Comment:
Should follow [property order standards](https://github.com/cfpb/front-end/blob/master/css.md#property-order), especially because there's so many set here.